projects/sandbox/src/app/_models/wizard-config.ts

```diff
@@ -0,0 +1,25 @@
+import { Validators } from '@angular/forms';
+
+export enum WizardFieldType {
+  select = 'select',
+  text = 'text',
+  progress = 'progress',
+  harvest = 'harvest'
+}
+
+export interface WizardField {
+  name: string;
+  label?: string;
+  type?: WizardFieldType;
```

Comment:
Yes: labeling can be ommited at field level and provided by the container, as with the file input.
Default values to an interface are not possible because interfaces only exists at compile time.